### PR TITLE
Move AndroidTableBuilder functionality to TableBuilder

### PR DESCRIPTION
**Pull request by wpride**
Remove AndroidTableBuilder

cross-request: https://github.com/dimagi/commcare-android/pull/1731

### CODE REVIEW

File: src/main/java/org/commcare/modern/database/TableBuilder.java [hunk 1]
```diff
@@ -189,4 +194,90 @@ public static String scrubName(String input) {
         }
         return bos.toByteArray();
     }
+
+    // TODO: Copied from AndroidTableBuilder, delete that when merged
```
Comment (amstone326) [inline src/main/java/org/commcare/modern/database/TableBuilder.java]: stale comment?

File: build.gradle [hunk 2]
```diff
@@ -65,6 +65,7 @@ dependencies {
       compile 'commons-cli:commons-cli:1.3.1'
       compile 'xpp3:xpp3:1.1.4c'
       compile 'joda-time:joda-time:2.9.3'
+      compile 'com.carrotsearch:hppc:0.7.2'
```
Comment (amstone326) [inline build.gradle]: what is this doing?

Comment (wpride) [inline build.gradle]: the `IntCollection` and `IntCursor` classes used by the sqlList functions require this dependency. Was already in `commcare-android`

**Comment by wpride** [comment]
@amstone326 I think because originally only Android used SQLite and moving things into `commcare-core` wasn't a priority

**Pull request merged**